reports/report_details.py

```diff
@@ -1496,6 +1522,188 @@ def add_transnational_to_regions(regions):
             'reports': ['global'],
         },
     },
+    'ws_98': {
+        '2015': {
+            'name': '98',
+            'title': 'Twitter - Images',
+            'desc': '',
+            'reports': [],
+        },
+        '2010': {
+            'name': '98',
+            'title': 'Twitter - Images',
+            'desc': '',
+            'reports': [],
+        },
+    },
+    'ws_100': {
+        '2015': {
+            'name': '100',
+            'title': 'Is this story related to Covid19. By major topic, by medium',
+            'desc': 'Is this story related to Covid19. By major topic, by medium',
+            'reports': [],
+        },
+        '2010': {
+            'name': '100',
+            'title': 'Is this story related to Covid19. By major topic, by medium',
+            'desc': 'Is this story related to Covid19. By major topic, by medium',
+            'reports': [],
+        },
+    },
+    'ws_101': {
+        '2015': {
+            'name': '101',
+            'title': 'Covid stories:  reporters, by sex',
+            'desc': 'Covid stories:  reporters, by sex',
+            'reports': ['region', 'country'],
+        },
+        '2010': {
+            'name': '101',
+            'title': 'Covid stories:  reporters, by sex',
+            'desc': 'Covid stories:  reporters, by sex',
+            'reports': ['region', 'country'],
+        },
+    },
+    'ws_102': {
+        '2015': {
+            'name': '102',
+            'title': 'Covid stories: gender stereotypes, by major topic',
+            'desc': 'Covid stories: gender stereotypes, by major topic',
+            'reports': ['region', 'country'],
+        },
+        '2010': {
+            'name': '102',
+            'title': 'Covid stories: gender stereotypes, by major topic',
+            'desc': 'Covid stories: gender stereotypes, by major topic',
+            'reports': ['region', 'country'],
+        },
+    },
+    'ws_103': {
+        '2015': {
+            'name': '103',
+            'title': 'Covid stories: highlight gender inequalities, by major topic',
+            'desc': 'Covid stories: highlight gender inequalities, by major topic',
+            'reports': ['region', 'country'],
+        },
+        '2010': {
+            'name': '103',
+            'title': 'Covid stories: highlight gender inequalities, by major topic',
+            'desc': 'Covid stories: highlight gender inequalities, by major topic',
+            'reports': ['region', 'country'],
+        },
+    },
+    'ws_104': {
```

Comment:
Hey @ascii-dev  @kilemensi 
For these new queries, seems like we forgot to add the corresponding report building functionality for them. This will/is causing the reports functionality to break.

Reply:
Hey @esirK ... We didn't forget but those are planned to be done next. @ascii-dev, did you test report generation when you added these?

As a temporary solution @esirK , lets add method that returns empty values if possible.